test.tf

```diff
@@ -108,24 +123,99 @@ resource "azurerm_postgresql_server" "example" {
   sku_name                     = "B_Gen5_2"
   storage_mb                   = 5120
   backup_retention_days        = 7
-  geo_redundant_backup_enabled = false
+  geo_redundant_backup_enabled = true
   auto_grow_enabled            = true
   administrator_login          = "DKFJnisdfu"
   administrator_login_password = "Aa12345678"
   version                      = "9.5"
   ssl_enforcement_enabled      = true
+
+  identity {
+    type = "SystemAssigned"
+  }
 }
 
 resource "azurerm_postgresql_configuration" "thrtottling_config" {
   name                = "connection_throttling"
   resource_group_name = "testrg"
   server_name         = azurerm_postgresql_server.example.name
-  value               = "off"
+  value               = "on"
 }
 
 resource "azurerm_postgresql_configuration" "example" {
   name                = "log_checkpoints"
   resource_group_name = "testrg"
   server_name         = azurerm_postgresql_server.example.name
-  value               = "off"
+  value               = "on"
+}
+data "azurerm_client_config" "example_clientconfig" {}
+resource "azurerm_postgresql_active_directory_administrator" "example_adconfig" {
+  server_name         = azurerm_postgresql_server.example.name
+  resource_group_name = azurerm_resource_group.resource_group.name
+  login               = "sqladmin"
+  tenant_id           = data.azurerm_client_config.example_clientconfig.tenant_id
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure that geo-redundant backups are enabled for Azure MySQL Server**
 
**Vulnerability**
Azure MySQL Server does not have geo-redundant backups enabled, this may lead to data loss.
 
**Remediation**
Ensure that your Azure MySQL Server have geo-redundant backups enabled, to allow you to restore your Azure MySQL Server to a different Azure region in the event of a regional outage or a disaster.

source: accurics-beta